src/components/layout/Layout.module.scss

```diff
@@ -0,0 +1,79 @@
+@use 'theme';
+
+.aminoLayout {
+  height: 100vh;
+  overflow: hidden;
+}
+
+.contentGrid {
+  height: theme.$amino-appbar-height;
```

Comment:
`sass` module allow to scope even more with declaring module before using any variable of that module to avoid overriding variable when importing scss variables with the same name.
Ex: we use `@use` for `theme` at top, so to use variable in the `theme`, you can use this syntax `theme.$variable`.